app/steps/medicaid/health_pregnancy.rb

```diff
@@ -2,5 +2,6 @@
 
 module Medicaid
   class HealthPregnancy < Step
+    step_attributes(:recently_pregnant)
```

Comment:
for `snap_applications` we call this `new_mom` because the actual question is about whether someone is pregnant or has been recently pregnant...thoughts on using that here, too?